### PR TITLE
Change logger to SeriLogger

### PR DESCRIPTION
**Pull request by nickmartin1ee7**
- Allows for future logging capabilities to file or metrics.
- Minor fixes with List command presentation.

### CODE REVIEW

File: Code2Gether-Discord-Bot.Library/BusinessLogic/ListProjectsLogic.cs [hunk 1]
```diff
@@ -39,29 +40,17 @@ private async Task<EmbedContent> ListProjectsAsync()
...
 
-            foreach (var project in projects)
+            for (var i = 0; i < projects.Count; i++)
             {
-
-                // Get Discord User details for project's author
-                var authorUser = await _context.Guild.GetUserAsync(project.Author.SnowflakeId);
-
-                sb.Append($"Project name: {project.Name}; Author: {authorUser.Username}#{authorUser.Discriminator}"
+                sb.Append($"[{i+1}/{projects.Count}]" + projects[i]
```
Comment (nickmartin1ee7) [inline Code2Gether-Discord-Bot.Library/BusinessLogic/ListProjectsLogic.cs:48]: The ToString override should suffice for this.

File: Code2Gether-Discord-Bot.Library/Models/Member.cs [hunk 2]
```diff
@@ -33,7 +33,7 @@ public Member(IUser user) : this()
         #endregion
 
         #region Methods
-        public override string ToString() => $"{SnowflakeId}";
+        public override string ToString() => $"<@{SnowflakeId}>";
```
Comment (nickmartin1ee7) [inline Code2Gether-Discord-Bot.Library/Models/Member.cs:36]: When rendered in a Discord Embed, it links to the user. Neat!

File: Code2Gether-Discord-Bot/Program.cs [hunk 3]
```diff
@@ -2,24 +2,27 @@
...
 namespace Code2Gether_Discord_Bot
 {
     class Program
     {
         static void Main(string[] args)
         {
-            ILogger logger = UtilityFactory.GetLogger(typeof(Program));
-            IBot bot = UtilityFactory.GetBot();
+            Log.Logger = new LoggerConfiguration()
+                .WriteTo.Console()
```
Comment (nickmartin1ee7) [inline Code2Gether-Discord-Bot/Program.cs:14]: This is where additional loggers are defined globally

**Review by DonSagiv — APPROVED** [review]
Don't know much about SeriLogger (I'm a log4net guy) but it looks good.

**Pull request merged**